DEVELOPER.md

```diff
@@ -214,14 +214,9 @@ To pull in a merge request you should generally follow the command line instruct
         $ git pull <remote_repo_url> <remote_branch>
     You can use `./dev-tools/storm-merge.py <pull-number>` to produce the above command most of the time.
 
-4.  Assuming that the pull request merges without any conflicts:
-    Update the top-level `CHANGELOG.md`, and add in the JIRA ticket number (example: `STORM-1234`) and ticket
-    description to the change log.  Make sure that you place the JIRA ticket number in the commit comments where
-    applicable.
+4. Run any sanity tests that you think are needed.
```

Comment:
The commit message refers to the commit on committing change on CHANGELOG.
If we would want to encourage contributors to place ticket number in the commit, it should be placed on `Create a pull request`. If you mean placing the ticket number on merge commit, we can describe it to 3.